core/src/main/scala/com/microsoft/azure/synapse/ml/core/utils/MapUtils.scala

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.core.utils
+
+object MapUtils {
+  def newMapArray(args: Seq[Seq[Any]]): Array[Map[Any, Any]] = {
```

Comment:
nit: perhaps name this as toSeqOfMaps and turn the Array type to Seq type

Reply:
Ah, this was a dead end. I'll delete it